docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,35 @@
+<style type="text/css">
+  .build-table  {border-collapse:collapse;border-spacing:0;}
+  .build-table th{font-weight: bold;}
+  .build-table th span.reduce-impact{font-weight: normal;}
+  .build-table span.code{color:#905;background-color:#DDD}
+  </style>
+  <table class="build-table">
+    <thead>
+      <tr>
+        <th colspan="2">magic-modules - build variables</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>OUTPUT_PATH <span class="reduce-impact">(required)</span> </th>
+        <td >The location of the provider you are unioning generation into.</td>
+      </tr>
+      <tr>
+        <th>VERSION <span class="reduce-impact">(required)</span></th>
+        <td>The version of the provider you are building into. <br/> Valid values are <span class="code">'ga'</span> and <span class="code">'beta'</span>.</td>
+      </tr>
+      <tr>
+        <th >ENGINE <span class="reduce-impact">(optional)</span></td>
+        <td >Modifies make provider to build either tpgtools or mmv1 valid values are <span class="code">'mmv1'</span> or <span class="code">`tpgtools`</span>. Note that running make provider with ENGINE set to <span class="code">`tpgtools`</span> will build any prerequisite  mmv1 files required for tpgtools.</td>
```

Comment:
There are extraneous code markers throughout - possibly left over from the markdown conversion? It would be great to clean up to avoid any confusion.

```suggestion
        <td >Modifies make provider to build either tpgtools or mmv1 valid values are <span class="code">mmv1</span> or <span class="code">tpgtools</span>. Note that running make provider with ENGINE set to <span class="code">tpgtools</span> will build any prerequisite  mmv1 files required for tpgtools.</td>
```